test_api.py

```diff
@@ -1829,7 +1829,7 @@ def _delete(self, url_query="", header=None):
             with patch("app.events.datetime", **{"utcnow.return_value": self.timestamp}):
                 url = f"{self.delete_url}{url_query}"
                 self.delete(url, 200, header, return_response_as_json=False)
-                return json.loads(m.events[0])
+                return json.loads(m.events[0][0])
```

Comment:
This starts to be a bit cryptic.

```suggestion
                event, key = m.events[0]
                return json.loads(event)
```